src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -118,6 +118,7 @@
 import org.apache.cassandra.db.SizeEstimatesRecorder;
 import org.apache.cassandra.db.SnapshotDetailsTabularData;
 import org.apache.cassandra.db.SystemKeyspace;
+import org.apache.cassandra.db.SystemKeyspace.BootstrapState;
```

Comment:
Do we really need this convenience?  We still have to import SystemKeyspace so it doesn't seem like much savings.

Reply:
sure, I can revert that.